isis/tests/TestCsmModel.cpp

```diff
@@ -607,14 +721,69 @@ csm::RasterGM::SensorPartials TestCsmModel::computeSensorPartials(
             double desiredPrecision,
             double* achievedPrecision,
             csm::WarningList* warnings) const {
-  
-  return csm::RasterGM::SensorPartials(0.0, 0.0);
+
+  double delta = 1.0;
+
+  // latitude/longitude
+  if (index == 2) {
+    delta = 0.5;
+  }
+  else {
+    // scale
+    delta = 0.0035;
+  }
+
+  std::vector<double> adj(getNumParameters(), 0.0);
```

Comment:
Would this benefit from a more descriptive name?

Reply:
Absolutely! Thanks for the suggestion